oak_functions/lookup_data_generator/src/data.rs

```diff
@@ -0,0 +1,125 @@
+//
+// Copyright 2021 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
```

Comment:
nit: Might be nice to add some documentation to the functions in this module. Also, seems like not all the functions need to be public. Perhaps make them private and only add doc comments to the ones that need to be public.

Reply:
Done.